.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: Build
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    container: devkitpro/devkitppc:latest
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - GAME: OOT
+          - GAME: MM
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+    # Checkout the repository (shallow clone)
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        submodules: recursive
+
+    # Set Git config
+    - name: Git config
+      run: git config --global --add safe.directory "$GITHUB_WORKSPACE"
+
+    # Build the project
+    - name: Build
+      run: make GAME=${{ matrix.GAME }}
```

Comment:
for MM could we build with `CPPFLAGS="-DHB_DBG -DHB_HEAP -DHB_FAT -DHB_N64VCMEM"`? this is what kz uses and otherwise tons of homeboy is never compiled here: https://github.com/krimtonz/kz/blob/30a965bc8f230f64d8c14b851d6f2707c2fcfe0b/makefile#L147

Reply:
I didn't realize it was missing a lot